xblock/fragment.py

```diff
@@ -1,269 +1,9 @@
-
-        `js_func` is the name of a Javascript function defined by one of the
-        Javascript resources.  As part of setting up the browser's runtime
-        environment, the function will be invoked, passing a runtime object
-        and a DOM element.
-
-        """
-        # This is version 1 of the interface.
-        self.js_init_fn = js_func
-        self.js_init_version = 1
-        if json_args:
-            self.json_init_args = json_args
-
-    # Implementation methods: don't override
-    # TODO: [rocha] should this go in the runtime?
-
-    def body_html(self):
-        """Get the body HTML for this Fragment.
-
-        Returns a Unicode string, the HTML content for the ``<body>`` section
-        of the page.
-
-        """
-        return self.content
-
-    def head_html(self):
-        """Get the head HTML for this Fragment.
-
-        Returns a Unicode string, the HTML content for the ``<head>`` section
-        of the page.
-
-        """
-        return self.resources_to_html("head")
-
-    def foot_html(self):
-        """Get the foot HTML for this Fragment.
-
-        Returns a Unicode string, the HTML content for the end of the
-        ``<body>`` section of the page.
-
-        """
-        return self.resources_to_html("foot")
-
-    def resources_to_html(self, placement):
-        """Get some resource HTML for this Fragment.
-
-        `placement` is "head" or "foot".
-
-        Returns a unicode string, the HTML for the head or foot of the page.
-
-        """
-        # TODO: [rocha] aggregate and wrap css and javascript.
-        # - non url js could be wrapped in an anonymous function
-        # - non url css could be rewritten to match the wrapper tag
-
-        return '\n'.join(
-            self.resource_to_html(resource)
-            for resource in self.resources
-            if resource.placement == placement
-        )
-
-    @staticmethod
-    def resource_to_html(resource):
-        """
-        Returns `resource` wrapped in the appropriate html tag for it's mimetype.
-        """
-        if resource.mimetype == "text/css":
-            if resource.kind == "text":
-                return u"<style type='text/css'>\n%s\n</style>" % resource.data
-            elif resource.kind == "url":
-                return u"<link rel='stylesheet' href='%s' type='text/css'>" % resource.data
-
-        elif resource.mimetype == "application/javascript":
-            if resource.kind == "text":
-                return u"<script>\n%s\n</script>" % resource.data
-            elif resource.kind == "url":
-                return u"<script src='%s' type='application/javascript'></script>" % resource.data
-
-        elif resource.mimetype == "text/html":
-            assert resource.kind == "text"
-            return resource.data
+from web_fragments.fragment import Fragment
 
-        else:
-            raise Exception("Never heard of mimetype %r" % resource.mimetype)
+__all__ = [
+    'Fragment',
+]
```

Comment:
There's no need for `__all__` here. It only affects "from xblock.fragment import *", and without it, the same names would be imported anyway.